src/unittest/python/aws_tests/ssm_tests.py

```diff
@@ -0,0 +1,14 @@
+try:
+    from unittest2 import TestCase
+    from mock import patch
+except ImportError:
+    from unittest import TestCase
+    from mock import patch
+from cfn_sphere.aws.ssm import SSM
+class SSMTests(TestCase):
```

Comment:
Please add spacing

Reply:
This is merged from upstream so if we change this we'll need to change this every time... :( 

(I would love to insert more spacing by the way)